src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --link-color: #000;
+  --link-active-color: #00acdc;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, 'Arial', sans-serif;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 5rem;
+  align-items: center;
+}
+
+.logo__link {
+  display: block;
+  width: 4rem;
+  height: 4rem;
+}
+
+.nav__list {
+  display: flex;
+  font-size: 1.2rem;
+  padding-inline-start: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin: 0 1rem;
```

Comment:
Be consistent with your vertical margins. Add only top or only bottom margin, but not both. It's better to choose one direction for vertical margins to avoid collapsing margins issues.